src/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -56,6 +57,13 @@
     
     abstract public SamReader open(final File file);
 
+    public SamReader open(final Path path) {
```

Comment:
We should probably add a test for this, since I don't think its covered elsewhere.